addons/website/controllers/main.py

```diff
@@ -776,6 +776,29 @@ def actions_server(self, path_or_xml_id_or_id, **post):
 
 
 class WebsiteBinary(http.Controller):
+
+    @http.route([
+        '/website/image',
+        '/website/image/<xmlid>',
+        '/website/image/<xmlid>/<int:width>x<int:height>',
+        '/website/image/<xmlid>/<field>',
+        '/website/image/<xmlid>/<field>/<int:width>x<int:height>',
+        '/website/image/<model>/<id>/<field>',
+        '/website/image/<model>/<id>/<field>/<int:width>x<int:height>'
+    ], type='http', auth="public", website=False, multilang=False)
+    # pylint: disable=redefined-builtin,invalid-name
```

Comment:
why?

Reply:
Because the `id` argument replaces the `id` builtin python function. `id` is part of the URL and that me have to manipulate it to extract `unique`, I cannot rename the argument as it would change the URL template and frankly `id` is a nice a concise name for an external API and one body ever uses the `id` function so it is safe of tell pylint to fuck off

Reply:
I meant that in the way that my linter is not raising any error without this and there is no occurrence of this in our whole code base (except for the real web content route), is this for runbot?

Reply:
Invoking pylint from the cli on my machine reports me the two problems:

    addons/website/controllers/main.py:793:36: C0103: Argument name "id" doesn't conform to snake_case naming style (invalid-name)
    addons/website/controllers/main.py:793:36: W0622: Redefining built-in 'id' (redefined-builtin)

and yes, ci/style would had reported the issue